jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebAppContext.java

```diff
@@ -1251,6 +1250,7 @@ protected void stopContext() throws Exception
 
             _unavailableException = null;
         }
+        super.stopContext();
```

Comment:
Since `super.stopContext()` must be the last thing done here, it's probably worth a comment.